hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/request/DefaultMetricAggregationRequestBuilder.java

```diff
@@ -85,7 +93,9 @@ private MetricAggregationRequest requestForAggregationField(
     return this.build(
         attribute,
         aggregationType,
-        this.getArgumentsForAggregation(aggregationType, field.getArguments()));
+        this.getArgumentsForAggregation(aggregationType, field.getArguments()),
+            this.selectionFinder.findSelections(field.getSelectionSet(),
+                    SelectionQuery.namedChild(BASELINE_AGGREGATION_VALUE)).collect(Collectors.toList()));
```

Comment:
combined with other comments, you could get a boolean out of this with `.findAny().isPresent()`